esmvalcore/cmor/_fixes/access/access_esm.py

```diff
@@ -0,0 +1,84 @@
+"""On-the-fly CMORizer for ACCESS-ESM.
+
+Note
+----
+This is the first version of ACCESS-ESM CMORizer in for ESMValCore
+Currently, only two variables (`tas`,`pr`) is fully supported.
+"""
+import logging
+
+from iris.cube import CubeList
+
+from esmvalcore.cmor._fixes.native_datasets import NativeDatasetFix
+
+logger = logging.getLogger(__name__)
+
+
+class AllVars(NativeDatasetFix):
+    """Fixes for all variables."""
+
+    def fix_coord_system(self, cube):
+        """Delete coord_system to make CubeList able to merge."""
+        for dim in cube.dim_coords:
+            if dim.coord_system is not None:
+                cube.coord(dim.standard_name).coord_system = None
+
+    def fix_height_value(self, cube):
+        """Fix height value to make it comparable to other dataset."""
+        if cube.coord('height').points[0] != 2:
+            cube.coord('height').points = [2]
```

Comment:
This looks slightly dangerous. There are variables like `uas` which use a scalar height of 10m. So it might be safer to put this in an explicit `class Tas(NativeDatasetFix)` fix.

Reply:
@rhaegar325 this has not been fixed. You should move it to the Tas class